📚More clarity added for Windows based Docker Desktop environments (#4068)
airbytehq/airbyte: docs/deploying-airbyte/local-deployment.md

We recommend following [this guide](https://docs.docker.com/docker-for-windows/install/) to install Docker on Windows. After installing the WSL 2 backend and Docker you should be able to run containers using Windows PowerShell. Additionally, as we note frequently, you will need `docker-compose` to build Airbyte from source. The suggested guide already installs `docker-compose` on Windows. 

### Handling long filename error

> If you are cloning the repo, you might run into a problem where git indicates that certain Filenames are too long and it therefore can't create the local file. Assuming repository has been cloned but you received long filename error, run the following command in *git bash*:

Reviewer: ```suggestion
If you are cloning the repo, you might run into a problem where git indicates that certain filenames are too long and it therefore can't create the local file. So if you received this error after cloning the repo, run the following commands in *git bash*:
```

```bash
cd airbyte
git config core.longpaths true
git reset --hard HEAD
```

> However it's worth pointing out that the above setting is defaulted to false for a reason, so use with caution. To localize the effect of this settings, git configuration is only changed within the cloned Airbyte repo. Find more details on [this stack overflow question](https://stackoverflow.com/questions/22575662/filename-too-long-in-git-for-windows).

Reviewer: ```suggestion
However it's worth pointing out that the `core.longpaths` option is defaulted to false for a reason, so use with caution. This git configuration is only changed within the cloned Airbyte repo, so you won't need to worry about changing this setting for other repositories. Find more details about this issue in [this stack overflow question](https://stackoverflow.com/questions/22575662/filename-too-long-in-git-for-windows).
```
